ConfCore/SessionInstance.swift

```diff
@@ -10,7 +10,7 @@ import Cocoa
 import RealmSwift
 
 public enum SessionInstanceType: Int {
-    case session, lab, video, getTogether, specialEvent
```

Comment:
I think you'll want to add it to the end, because the type is stored in a Realm as the `rawValue`, which means for existing db's all the sessions would theoretically get new types.

Reply:
I don't think that's an issue since when merging the content from the feed, the types will be overridden by whatever comes from the feed. I'll do that anyway just to be extra careful.